docs/.theme/components/Header.vue

```diff
@@ -45,8 +45,6 @@
           <svg
             viewBox="0 0 32 32"
             xmlns="http://www.w3.org/2000/svg"
-            style="
-"
```

Comment:
let's make this svg a component `components/Icon/Matrix.vue` and use it here `<Icon.Matrix />`